TestResultSummaryService/routes/getBenchmarkVariants.js

```diff
@@ -0,0 +1,5 @@
+const BenchmarkMetric = require( '../parsers/BenchmarkMetric');
```

Comment:
Maybe we should rename the file to `getBenchmarkParsingInfo` since that's more reflective since we are also sending the router info.